GameData/Kerbalism/Profiles/Default.cfg

```diff
@@ -289,7 +288,7 @@ Profile
   {
     name = atmo leaks
     modifier = surface,breathable
-    input = Atmosphere@0.000000148
+    input = Atmosphere@0.000148
     // from ISS: 899 m³ volume, 452 m² surface (estimated), 0.227 Kg/day (structural) + 1.543 Kg/day (activities)
```

Comment:
how is this input related to this comment? (considering 3600*6 seconds per day)

Reply:
oh wait he disappeared

Reply:
lol